### DISCUSSION TITLE
How to train XTTS_v2 with a dataset that is not 22050Hz sample rate?

### DISCUSSION THREAD

**Discussion by jafarmahrook** [coqui-ai/TTS#3578] [General Q&A]
How can I train a XTTS_v2 model with a dataset of 44100 Hz wav audio files and get the result with the same sample rate 44100Hz?

**Comment by jafarmahrook** [comment]
@Edresson

**Comment by gougang** [comment]
同问，希望大神给解答一下，谢谢

**Comment by jafarmahrook** [comment]
@erogol
